cloud/aws/aws_file_system.cc

```diff
@@ -33,6 +33,31 @@
 
 namespace ROCKSDB_NAMESPACE {
 
+std::shared_ptr<void> useAWS() {
```

Comment:
Do we need to update `AwsFileSystem::AwsFileSystem` as well? Should it call `useAWS` as well?

Looking at the [AWS SDK API](https://docs.aws.amazon.com/sdk-for-cpp/v1/developer-guide/basic-use.html),  I'm in favor for the following:
* `AwsFileSystem::AwsFileSystem` doesn't initialize aws sdk and requires users to setup already
* have rocksdb-cloud consumers manage `Aws::InitAPI` lifecycle. It also allows them to configure `SDKOptions`
* This offers `useAWS` as a helper for initialization to ease initialization if consumers want to have fine grain/scoped AWS clients - They can alternatively call `Aws::InitAPI`/`Shutdown` in their main?

What do you think?

Reply:
I assume for the most parts, users will call `InitAPI` in main once and Shutdown at the end on exit.